wgsl/index.bs

```diff
@@ -2669,6 +2669,10 @@ It uses a two's complementation representation, with the sign bit in the most si
     <tr><td>0x0u<td>0xffffffffu
 </table>
 
+The <dfn>u64</dfn> type is the set of 64-bit unsigned integers.
```

Comment:
I'm unhappy with how this names `u64` just like the other integer types.

"Note"s aren't supposed to be normative. 
I would rather do something like what's done for f64 in the [Floating Point Conversion](https://gpuweb.github.io/gpuweb/wgsl/#floating-point-conversion) section:

> A hypothetical type corresponding to a binary format defined by the [IEEE-754](https://gpuweb.github.io/gpuweb/wgsl/#biblio-ieee-754) floating point standard.

I suggest removing this section, and modifying the later block...

Reply:
Makes sense. Done!